inc/Engine/Admin/RocketInsights/Jobs/Manager.php

```diff
@@ -230,6 +249,63 @@ private function parse_test_results( array $api_response ): array {
 		return wp_parse_args( $api_response['data']['data'], $defaults );
 	}
 
+	/**
+	 * Attempt synchronous submission to GTMetrix API.
+	 *
+	 * @param string $url URL to test.
+	 * @param bool   $is_mobile Is mobile.
+	 * @param array  $additional_details Additional details.
+	 * @return array|false Returns job data on success, false on failure/timeout.
```

Comment:
We should use WP_Error instead of `false` on failure. This probably should be extended to the `add_to_queue()` method as well